packages/wonder-blocks-modal/components/modal-dialog.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {View} from "wonder-blocks-core";
+
+import typeof ModalContentPane from "./modal-content-pane.js";
+
+export default class ModalDialog extends React.Component<{
+    children: React.ChildrenArray<React.Element<
+        ModalContentPane,
+    > | null | void>,
```

Comment:
I'm not sure what prettier was thinking here.  Can you pull these props out into a separate type declaration?

Reply:
Ha, will do.